db/postgres/postgres_types.go

```diff
@@ -202,3 +204,19 @@ type Compatibility struct {
 	State string
 	Note  string
 }
+
+type ModLink struct {
+	ID               string `gorm:"primary_key;type:varchar(16)"`
+	ModVersionLinkID string
+	Platform         string
+	Key              string
+	Size             int64
+	Hash             string
+}
+
+type SMLLink struct {
```

Comment:
Please rename to `SMLArch`

Reply:
https://github.com/satisfactorymodding/smr-api/pull/12/commits/bdd565d8997568955b8e33bcb352c38c7af5c2c5